proxy/Main.cc

```diff
@@ -1507,9 +1507,8 @@ main(int /* argc ATS_UNUSED */, const char **argv)
   // re-start it again, TS will crash.
   // This is also needed for log rotation - setting up the file can cause privilege
   // related errors and if diagsConfig isn't get up yet that will crash on a NULL pointer.
-  diagsConfig       = new DiagsConfig(DIAGS_LOG_FILENAME, error_tags, action_tags, false);
-  diags             = diagsConfig->diags;
-  diags->prefix_str = "Server ";
+  diagsConfig = new DiagsConfig("Server ", DIAGS_LOG_FILENAME, error_tags, action_tags, false);
```

Comment:
I favor removing the spaces from `prefix_str` and putting it in the formatting logic. I suspect it was done this way originally to allow not having a prefix but I think we can dispose of that.